js/packages/teams-ai/src/planners/AssistantsPlanner.ts

```diff
@@ -0,0 +1,309 @@
+import OpenAI from 'openai';
+import { Planner, Plan, PredictedDoCommand, PredictedSayCommand } from './Planner';
+import { TurnState } from '../TurnState';
+import { TurnContext } from 'botbuilder';
+import { AI } from '../AI';
+
+const DEFAULT_POLLING_INTERVAL = 1000;
+const DEFAULT_ASSISTANTS_STATE_VARIABLE = 'conversation.assistants_state';
+const SUBMIT_TOOL_OUTPUTS_VARIABLE = 'temp.submit_tool_outputs';
+const SUBMIT_TOOL_OUTPUTS_MAP = 'temp.submit_tool_map';
+
+export interface AssistantsPlannerOptions {
+    apiKey: string;
+    assistant_id: string;
+    polling_interval?: number;
+    assistants_state_variable?: string;
+}
+
+export class AssistantsPlanner<TState extends TurnState = TurnState> implements Planner<TState> {
```

Comment:
Please add docs when you can - fast follow is no problem. Summary of AssistantsPlanner would be great (like for the other modules)

Reply:
I'll fast follow with this